os/net/lwip/src/netif/etharp.c

```diff
@@ -72,7 +72,7 @@
 #include <net/lwip/dhcp.h>
 #include <net/lwip/ipv4/autoip.h>
 #include <net/lwip/netif/etharp.h>
-
+#include <arpa/inet.h>
```

Comment:
Why include the header file here, if the rest of the file does not use it? Isn't it better to include files where they are used directly?

Reply:
After adding Werror flag we were getting htons related error to overcome that error we have added this header file.